NEWS.md

```diff
@@ -1,5 +1,12 @@
 # tidyr (development version)
 
+## Bug fixes and minor improvements
```

Comment:
Can you please remove this? We only add just prior to release (https://style.tidyverse.org/news.html#news-release)